library/src/skins/default/src/sass/base/_icons.scss

```diff
@@ -170,6 +170,7 @@ $fa-font-path: "./fonts";
     remove: bi-x-square-fill,
     empty-square: bi-square,
     check-square: bi-check-square,
+    feedbackCheck-Samigo: bi-check-lg,
```

Comment:
I'd just make this check-lg or something. It doesn't seem right to make this samigo specific. Other tools may want to use it, in fact we want to encourage that. Also, I don't think we use camel case anywhere in the other icon names.

Reply:
Good point, I will make the changes and let you know :)